### ISSUE TITLE
BrowserWindow's webPreferences.nodeIntegration not work

### ISSUE THREAD

**Issue by ppoo24** [atom/electron#3579]
On an Render Process Window, i want to  open a new window without Node:
~~~js
var BrowserWindow = require('remote').require('browser-window');
var testWin = new BrowserWindow({width:650, height:480,webPreferences: { nodeIntegration: false }}); testWin.loadUrl('http://www.baidu.com');
testWin.webContents.openDevTools();
~~~
then i checked in console of the new window's DevTool, it still has the rights to access `node`(for example: successfully visiting the `require` function).
Don't know why

**Comment by ppoo24** [comment]
Okay! I just learn from https://github.com/atom/electron/blob/master/docs/api/browser-window.md, it shows the key in BrowserWindow's `options` do use camel-case naming, but actually using delimiter '-' is the right way.
Still Don't know why using that style for doc?

**Comment by zcbenz** [comment]
The `webPreferences.nodeIntegration` style is only available after v0.35.x, you should view the documents that match the version of Electron you use. The [README of docs](https://github.com/atom/electron/tree/master/docs) explained this.

**Comment by ppoo24** [comment]
sorry for my carelessness, the electron-prebuilt in npm's registry is behind the HEAD version
